src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,183 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.util.*;
+
+/**
+ * Takes in and processes user input according to the available bot command.
+ */
+public class Parser {
+    private TaskList myTasks;
+
+    public Parser(Scanner scanner, TaskList myTasks) {
+        this.myTasks = myTasks;
+    }
+
+    /**
+     * Processes a single command.
+     *
+     * @param command taken in by scanner
+     * @return boolean that denotes whether to continue processing
+     */
+    public boolean processCmd(String command) {
```

Comment:
Great naming of boolean members and methods. 👍 

Could the `processCmd` method possibly be named to sound more like a boolean or adjust it such that `processCmd` returns the type of command and the main `Duke` class or some other handles the execution of the commands.